tests/PHPStan/Parser/TypeParserTest.php

```diff
@@ -393,7 +393,12 @@ public function provideParseData(): array
 				]),
 			],
 			[
-				'array{a: int, b: array{c: callable(): int}}',
+				'array{
+				    a: int, 
+				    b: array{
+				        c: callable(): int
+				    }
+				 }',
```

Comment:
You should add new test instead of modifying an existing one

Reply:
Also the test is incorrect, as it's missing the `*`. What you are actually parsing is sth like

```
array{
 * a: int,
 * b: array{
 *   c: callable(): int
 * }
 * }
```

Reply:
Is it all right if I just add a single test showing that the leading `*`s don't disrupt anything? I'd otherwise like to omit them for readability. 414daf3